crypto/fipsmodule/ec/p384.c

```diff
@@ -503,6 +503,479 @@ static int ec_GFp_nistp384_cmp_x_coordinate(const EC_GROUP *group,
   return 0;
 }
 
+//                    SCALAR MULTIPLICATION OPERATIONS
+// ----------------------------------------------------------------------------
```

Comment:
I think we need a reference like the one suggested in make_table.go. It should also outline where we deviate from the reference. But this can wait until we discuss the analysis of the recoding.